generator/views/glean_ping_view.py

```diff
@@ -63,11 +63,13 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
                 looker_name = self._to_looker_name(metric)
                 view_name = f"{self.name}__{looker_name}"
                 suggest_name = f"suggest__{view_name}"
+
+                category, name = self._get_category_and_name(metric)
+                view_label = f"{category.replace('_', ' ').title()} - {name.replace('_', ' ').title()}"
```

Comment:
I think I mildly prefer a list comprehension here (take this suggestion or leave it, not a big deal either way)

```suggestion
                category, name = [v.replace('_', ' ').title() for v in self._get_category_and_name(metric)]
                view_label = f"{category} - {name}"
```

Reply:
I do as well, reduced duplication without requiring another function. r+ :)